### PR TITLE
AAP-32403: Lightspeed: Playbook generation: Handle "warnings" in response

### PR DESCRIPTION
**Pull request by manstis**
VSCode changes to support display of "warnings" resulting from Playbook Generation requests.

### CODE REVIEW

**Comment by manstis** [comment]
SonarCloud complaints are about duplicate test code.. 

Personally, the duplication makes individual tests easier to read.

File: src/features/lightspeed/playbookGeneration.ts [hunk 1]
```diff
@@ -332,6 +339,23 @@ export function getWebviewContent(webview: Webview, extensionUri: Uri) {
                 <li></li>
               </ol>
             </div>
+            <div class="warningsContainer">
```
Comment (TamiTakamiya) [inline src/features/lightspeed/playbookGeneration.ts:342]: This implementation looks very nice. Thanks for the efforts on this.  

Followng is just an informative comment:

So far th errors found in Playbook Generation are shown using VS Code's showErrorMessage API ([example](https://github.com/ansible/vscode-ansible/blob/main/src/features/lightspeed/playbookGeneration.ts#L180-L182)), which opens a small pop-up window at the right-bottom of the workbench. I believe it is a standard way and nothing wrong, but showing an error/warning message in the Playbook Generation UI would be more intuitive (btw, Playbook Explanation also displays error messages in its own window). I understand this PR won't be merged into the main branch for now, but when it it is merged to main, we want to re-consider the way to display error messages occured in Playbook Generation.

File: test/mockLightspeedServer/generations.ts [hunk 2]
```diff
@@ -118,9 +119,17 @@ export function generations(
     outline += `\n6. model=${model}`;
   }
 
+  let warnings: GenerationResponseWarning[] = [];
+  if (text === "Create a playbook with warnings") {
```
Comment (TamiTakamiya) [inline test/mockLightspeedServer/generations.ts:123]: :smile:

**Review by TamiTakamiya — APPROVED** [review]
LGTM. I left a comment. It's just informative and no action is required for that. Thanks!

File: src/features/lightspeed/playbookGeneration.ts (hunk 1 above)
Comment (manstis) [reply src/features/lightspeed/playbookGeneration.ts:342]: Ah yes indeed @TamiTakamiya 

The "warnings" however are not errors...

The WCA call returns with HTTP200, a `playbook` and _sometimes_ an `outline` and _sometimes_ a `warnings` list.

It is these `warnings` I am propagating to the UI.. Playbook generation can proceed but may give unexpect results.

Comment (TamiTakamiya) [reply src/features/lightspeed/playbookGeneration.ts:342]: jfyi. VS Code API has [showWarningMessage](https://code.visualstudio.com/api/references/vscode-api#window.showWarningMessage).  Though I am not a big fan of those APIs, it's an easy route...

**Pull request merged**